src/site/content/en/blog/ta-code-coverage/index.md

```diff
@@ -150,7 +160,7 @@ In the code example, there are two functions: `calcCoffeeIngredient` and `isVali
 
 **Code coverage: 62.5%**
 
-```js/5-7,10,15/11-12,19
+```text/5-7,10,15/11-12,19
```

Comment:
purposely use text instead of js here and other places to show the highlighted code more, especially in dark mode.